packages/graphql/src/modules/adoption/AdoptionQuery.ts

```diff
@@ -0,0 +1,28 @@
+import {GraphQLFieldConfigMap, GraphQLList} from 'graphql';
+import {connectionDefinitions, forwardConnectionArgs, connectionFromPromisedArray} from 'graphql-relay';
+
+import {GraphQLContext} from '../../types';
+
+import AdoptModel from './AdoptionModel';
+import AdoptionType from './AdoptionType';
+
+export const AdoptionConnection = connectionDefinitions({
+  name: 'Adoption',
+  nodeType: AdoptionType,
+});
+
+export const AdoptionsQueries: GraphQLFieldConfigMap<any, GraphQLContext, any> = {
+  adoptions: {
+    type: AdoptionConnection.connectionType,
+    args: forwardConnectionArgs,
+    resolve: (_, args) => connectionFromPromisedArray(AdoptModel.find().populate('user') as any, args),
```

Comment:
IMO I think it's better to move these resolvers to another file. They can get big over time and will be hard to read having them inline.